components/DataView.vue

```diff
@@ -39,7 +39,9 @@
           <div>
             <a class="Permalink" :href="permalink()">
               <time :datetime="formattedDate">
-                {{ $t('{date} 更新', { date }) }}
+                {{
+                  $t('{formattedDateByLocale} 更新', { formattedDateByLocale })
```

Comment:
こちらですが、
```suggestion
                  $t('{date} 更新', { date: formattedDateByLocale })
```
というように変数に代入できるため、 `{date}`の部分はそのままにしておきたいです（翻訳者側にわかりやすくするため）

Reply:
横から失礼します。
今後言語が増えることを考慮するのであれば、言語をキーにしたformat文字列のオブジェクトを用意するのではなく、
`dayjs(dateString).format('LLL')`
とすれば今後修正が不要になって楽かと思いますがいかがでしょう。
任意の表記で調整することはできませんが。。